travis.gemspec

```diff
@@ -299,7 +299,9 @@ Gem::Specification.new do |s|
   s.add_dependency "launchy",               "~> 2.1"
   s.add_dependency "typhoeus",              "~> 0.6", ">= 0.6.8"
   s.add_dependency "pusher-client",         "~> 0.4"
-  s.add_dependency "addressable",           "~> 2.3"
+  if Gem::Version.new(RUBY_VERSION) < Gem::Version.new("1.9.0")
```

Comment:
>  Ah, I see. We are still running builds on 1.8.7. We should probably fix that.

Nice. 1.9.3 or greater makes things much easier. I can add that change in this PR if it makes sense (any other PR is going to fail the build until the issue with addressable is resolved anyways).